lua/net.lua

```diff
@@ -101,6 +101,10 @@ return function (gatekeeper_server)
 	net_conf.log_level = log_level
 	net_conf.rotate_log_interval_sec = rotate_log_interval_sec
 
+	if back_iface_enabled then
+		staticlib.check_ifaces(front_ports, back_ports)
```

Comment:
Another check that may help in production to make sure that `front_ports` (and `back_ports`) does not contain duplicates. Could you verify if the error Gatekeeper throws when there're duplicates is good enough to help users? If not, could you extend this patch to verify it in Lua as well?

Reply:
When this happens, an error occurs in the bonded port library:

```
__eth_bond_slave_add_lock_free(458) - Slave device is already a slave of a bonded device
GATEKEEPER: net: failed to add slave port 4 to bonded port 6 (err=-1)
```

I don't think this is entirely clear so I will adjust the patch to check for this as well.